### ISSUE TITLE
Quickstart Terminal does not support fish shell

### ISSUE THREAD

**Issue by victorhooi** [docker/toolbox#185]
I've installed Docker Toolbox 1.8.2 on OSX 10.11 (El Capitan GM).

If I try to startup the Docker Quickstart Terminal, I get a prompt asking me to pick between iTerm and Terminal. If I pick iTerm, I get:

<img width="430" alt="screenshot 2015-09-15 19 29 38" src="https://cloud.githubusercontent.com/assets/243499/9872983/2b69f14c-5be0-11e5-92d9-a0a3a0ce70df.png">

If I pick Terminal I get a Terminal with the following output:

```
Last login: Tue Sep 15 18:52:50 on ttys012
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
victorhooi@thadeus ~> /usr/local/bin/fish --login
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
victorhooi@thadeus ~> . '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
Unsupported use of '||'. In fish, please use 'COMMAND; or COMMAND'.
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh (line 16): if [ ! -f $DOCKER_MACHINE ] || [ ! -f $VBOXMANAGE ]; then
                                                                                                                                ^
from sourcing file /Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh
	called on line 151 of file /usr/local/Cellar/fish/HEAD/share/fish/config.fish

in function '.'
	called on standard input

source: Error while reading file '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
```

If I try to run any Docker commands, I get:
```
victorhooi@thadeus ~> docker ps
Get http:///var/run/docker.sock/v1.20/containers/json: dial unix /var/run/docker.sock: no such file or directory.
* Are you trying to connect to a TLS-enabled daemon without TLS?
* Is your docker daemon up and running?
```

I also tried to startup Kitematic. Once it's started up, if I click on the "Docker CLI" button in the bottom left, I get a Terminal window with two tabs (not sure why there's two tabs):

<img width="580" alt="screenshot 2015-09-15 19 32 39" src="https://cloud.githubusercontent.com/assets/243499/9873032/8ea64b0c-5be0-11e5-8f65-107801845e42.png">

If I try to run `docker ps` in here, I also get:
```
victorhooi@thadeus ~> docker ps
An error occurred trying to connect: Get https:///var/run/docker.sock/v1.20/containers/json: dial unix /var/run/docker.sock: no such file or directory
```

**Comment by mchiang0610** [comment]
@victorhooi It seems like your Docker machine instance was not able to install correctly. 

Can you go into Virtualbox and remove the ```default``` VM by using "Delete All Files". Run the toolbox installer again.

**Comment by jeffdm** [comment]
It seems that fish shell is not supported, marking as a bug.

**Comment by jeffdm** [comment]
After some testing, this should be fixed in [1.8.2a](https://github.com/docker/toolbox/releases/tag/v1.8.2a). Thanks and feel free to re-open if not.
